matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/mapper/EventAnnotationsSummaryMapper.kt

```diff
@@ -36,13 +37,22 @@ internal object EventAnnotationsSummaryMapper {
                     )
                 },
                 editSummary = annotationsSummary.editSummary
-                        ?.let {
-                            val latestEdition = it.editions.maxByOrNull { editionOfEvent -> editionOfEvent.timestamp } ?: return@let null
+                        ?.let { summary ->
```

Comment:
I think we should take the occasion to introduce a dedicated `EditAggregatedSummaryMapper` (as a classical class and not an `Object`). We could then add unit tests on it.

Reply:
Extracted as object and added tests